python/lsst/rubintv/production/utils.py

```diff
@@ -210,12 +213,21 @@ def remakeDay(channel, dayObs, remakeExisting=False, notebook=True, **kwargs):
     ------
     ValueError:
         Raised if the channel is unknown.
+        Raised if remakeExisting is False and channel is auxtel_metadata
     """
+    if not logger:
+        logger = logging.getLogger(__name__)
+
     from google.cloud import storage
 
     if channel not in CHANNELS:
         raise ValueError(f"Channel {channel} not in {CHANNELS}")
 
+    if remakeExisting and channel in ['auxtel_metadata']:
```

Comment:
Am I thinking about this logic wrong?  If `remakeExisting = True`, then this block will trigger, but it claims that `remakeExisting = True` will avoid this?

Reply:
Nope, you are 100% correct, that logic is backwards and the docs are correct. Fixed to `if remakeExisting is False...`